Design.md

```diff
@@ -21,12 +26,12 @@ Container Linux installer; the core of it will be dd'ing an image to the disk.
 
 The partition layout is still undecided, but initial proposals look something like:
 
-    Number	Type	Purpose
+    Number  Type    Purpose
     -----------------------------------
-    1	fat32	Boot partition/ESP
```

Comment:
but I like tabs :(

Reply:
oh man haha.. do you really want me to change it back?

Reply:
I mean kinda. Not trying to start a holy war, but tabs work nicely for tables.

Reply:
I fixed it back to what you had before but no matter what I do I can't make it line up correctly in VIM. I set tabstop to 4 and 8 and neither one of them gives me what I would expect.